README.md

```diff
@@ -1,27 +1,28 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/morningconsult/serrors)](https://goreportcard.com/report/github.com/morningconsult/serrors)
 # serrors
 
-serrors is a collection of three error implementations. `StackErr` provides call-stack information,
-based on ideas from https://github.com/pkg/errors . `Sentinel` creates immutable package-level
-sentinel errors, based on ideas from Dave Cheney. `StatusError` includes error category info alongside
-the source error.
+serrors is a collection of utitilies for creating and working with errors.
+These include automatic stacktraces based on ideas from
+<https://github.com/pkg/errors>, immutable package-level sentinel errors,
+based on ideas from Dave Cheney, and the ability to combine HTTP statuses with
```

Comment:
Question: does this need a citation or anything?

Reply:
I'm assuming it's a reference to this, but I'm not 100% sure: https://dave.cheney.net/2016/04/27/dont-just-check-errors-handle-them-gracefully

I don't really know that a citation is necessary or relevant here. Up to you if you think a link would make our docs better.

Reply:
Nah, if you are not sure then I wouldn't cite anything. I only mentioned it because I think its better to cite someone when you are claiming they said something, but no big deal.